api/app/controllers/spree/api/base_controller.rb

```diff
@@ -2,170 +2,176 @@
 
 module Spree
   module Api
-    module V1
-      class BaseController < ActionController::Base
-        include Spree::Api::ControllerSetup
-        include Spree::Core::ControllerHelpers::Store
-        include Spree::Core::ControllerHelpers::StrongParameters
-
-        attr_accessor :current_api_user
-
-        class_attribute :error_notifier
-
-        before_action :set_content_type
-        before_action :load_user
-        before_action :authorize_for_order, if: Proc.new { order_token.present? }
-        before_action :authenticate_user
-        before_action :load_user_roles
-
-        rescue_from Exception, with: :error_during_processing
-        rescue_from ActiveRecord::RecordNotFound, with: :not_found
-        rescue_from CanCan::AccessDenied, with: :unauthorized
-        rescue_from Spree::Core::GatewayError, with: :gateway_error
-
-        helper Spree::Api::ApiHelpers
-
-        def map_nested_attributes_keys(klass, attributes)
-          nested_keys = klass.nested_attributes_options.keys
-          attributes.inject({}) do |h, (k,v)|
-            key = nested_keys.include?(k.to_sym) ? "#{k}_attributes" : k
-            h[key] = v
-            h
-          end.with_indifferent_access
-        end
+    class BaseController < ActionController::Base
+      include Spree::Api::ControllerSetup
+      include Spree::Core::ControllerHelpers::Store
+      include Spree::Core::ControllerHelpers::StrongParameters
 
-        # users should be able to set price when importing orders via api
-        def permitted_line_item_attributes
-          if @current_user_roles.include?("admin")
-            super + [:price, :variant_id, :sku]
-          else
-            super
-          end
-        end
+      attr_accessor :current_api_user
 
-        private
+      class_attribute :error_notifier
 
-        def set_content_type
-          content_type = case params[:format]
-          when "json"
-            "application/json; charset=utf-8"
-          when "xml"
-            "text/xml; charset=utf-8"
-          end
-          headers["Content-Type"] = content_type
-        end
+      before_action :set_content_type
+      before_action :load_user
+      before_action :authorize_for_order, if: Proc.new { order_token.present? }
+      before_action :authenticate_user
+      before_action :load_user_roles
 
-        def load_user
-          @current_api_user = Spree.user_class.find_by(spree_api_key: api_key.to_s)
-        end
+      after_filter  :set_jsonp_format
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.